src/components/Loading/Loading.jsx

```diff
@@ -1,10 +1,11 @@
-import './Loading.module.css';
+import "./Loading.module.css";
 
 export const Loading = () => (
-    <div className="bg-purple-200">
-        <span className="loading loading-spinner loading-xs fuchsia-600"></span>
-        <span className="loading loading-spinner loading-sm fuchsia-600"></span>
-        <span className="loading loading-spinner loading-md fuchsia-600"></span>
-        <span className="loading loading-spinner loading-lg fuchsia-600"></span>
-    </div>
-)
\ No newline at end of file
+  // Стоит наверное выбрать один loading spinner size
```

Comment:
![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/bad542bd-4e62-4922-9786-939e2803148c)

Reply:
This Loading spinner was taken as a ready component from the DaisyUI, but for the better customer approach, I changed it. Done.